Lib/javascript/v8/javascriptrun.swg

```diff
@@ -263,43 +297,52 @@ SWIGRUNTIME int SWIG_V8_ConvertPtr(SWIGV8_VALUE valRef, void **ptr, swig_type_in
   return SWIG_V8_ConvertInstancePtr(objRef, ptr, info, flags);
 }
 
-SWIGRUNTIME SWIGV8_VALUE SWIG_V8_NewPointerObj(void *ptr, swig_type_info *info, int flags) {
+SWIGRUNTIME SWIGV8_VALUE SWIG_V8_NewPointerObj(void *ptr, swig_type_info *info, int flags,
+                                               SWIGV8_VALUE data) {
+  v8::Isolate* isolate = v8::Isolate::GetCurrent();
   SWIGV8_HANDLESCOPE_ESC();
-  
+  v8::Persistent<v8::FunctionTemplate>* v8_swig_types = NULL;
+#if (V8_MAJOR_VERSION-0) >= 5
+  v8::Local<v8::ArrayBuffer> ab = v8::Local<v8::ArrayBuffer>::Cast(data);
+#if (V8_MAJOR_VERSION-0) >= 8
```

Comment:
`#if (V8_MAJOR_VERSION-0) >= 5` can be assumed now.  The `>= 8` check should be `SWIG_V8_VERSION >= 0x0800` to match existing code.

And similarly for any other uses to avoid me having to note them individually when a `git grep V8_MAJOR_VERSION` will find them for you.

Reply:
Fixed.